src/reducers/ui.js

```diff
@@ -24,6 +24,29 @@ const defaultState = new Immutable.Map().
   set('topBar', new Immutable.Map({openMenu: null})).
   set('lastRefreshTimestamp', null);
 
+// i.e. restore to before editors were resized
+function restoreDefaultColumnFlex(state) {
+  return state.setIn(['workspace', 'columnFlex'], DEFAULT_COLUMN_FLEX);
+}
+
+function restoreDefaultRowFlex(state) {
+  return state.setIn(['workspace', 'rowFlex'], DEFAULT_ROW_FLEX);
+}
+
+function restoreFlexOnComponentToggle(componentName, state) {
+  if (componentName === 'output') {
+    return restoreDefaultRowFlex(state);
```

Comment:
It doesn't seem like the output component is ever hidden, but left this in here in case that's part of the roadmap. Happy to delete this functionality and simplify if that would be better!

Reply:
Yep, you’re right about that—it actually used to be possible, but the redesign didn’t have an output-minimization button, so I went ahead and dropped it from the initial implementation of the new layout. I think we could probably just put one in the preview top bar (next to either the refresh or the pop-out button) but also no one has mentioned missing it…